src/main/scala/com/atomist/rug/runtime/js/JavaScriptResponseHandler.scala

```diff
@@ -19,7 +20,20 @@ class JavaScriptResponseHandlerFinder
   override val kind: String = "response-handler"
 
   override protected def extractHandler(jsc: JavaScriptContext, handler: ScriptObjectMirror, externalContext: Seq[AddressableRug]): Option[JavaScriptResponseHandler] = {
-    Some(new JavaScriptResponseHandler(jsc, handler, name(handler), description(handler), parameters(handler), tags(handler), externalContext))
+    Some(new JavaScriptResponseHandler(jsc, handler, name(handler), description(handler), parameters(handler), tags(handler), coerce(jsc,handler), externalContext))
+  }
+
+  /**
+    * Figure out if we need to parse json or whatever
+    * @param handler
+    * @return
+    */
+  def coerce(jsc: JavaScriptContext, handler: ScriptObjectMirror): ResponseCoercer = {
+    handler.getMember("__coercion") match {
+      case "JSON" => new JsonResponseCoercer(jsc, handler)
+      case str: String => throw new InvalidHandlerException(s"Don't know how to coerce responses to $str")
```

Comment:
The `__coercion` can only take `JSON` as a value right now, and failing fast if folk get this wrong could be helpful for debugging.